jetstream/tests/test_config.py

```diff
@@ -381,14 +381,14 @@ def test_silly_query(self, experiments):
 
     def test_control_branch(self, experiments):
         trivial = config.AnalysisSpec().resolve(experiments[0])
-        assert trivial.experiment.control_branch == "b"
+        assert trivial.experiment.reference_branch == "b"
 
         conf = dedent(
             """
             [experiment]
-            control_branch = "a"
+            reference_branch = "a"
```

Comment:
Is it okay to simply rename this?

Reply:
Yeah, this is about our last chance to make a change like this with no consequences :p